SrProj/Static/partials/patronCheckIn.html

```diff
@@ -0,0 +1,142 @@
+                        <option value="3">Black or African American</option>
+                        <option value="2">Hispanic or Latino</option>
+                        <option value="1">Native Hawaiian or Other Pacfic Islander</option>
+                        <option value="0">White</option>
+                    </select>
+                </div>
+                </div>
+                <!--Adress & Postal code-->
+                <h4 class="ui dividing header">Address</h4>
+                <div class="two fields">
+                   
+                    <div class="field">
+                        <input type="text" placeholder="Street Address" />
+                    </div>
+                    <div class="field">   
+                        <input type="text" placeholder="Apt #" />
+                    </div>
+                    <div class="field">
+                        <input type="text" placeholder="Postal Code" />
+                    </div>
+                    <div class="field">
+                        <select class="ui dropdown">
+                            <option value="">Household Occupants</option>
+                            <option value="7">1</option>
+                            <option value="6">2</option>
+                            <option value="5">3</option>
+                            <option value="4">4</option>
+                            <option value="3">5</option>
+                            <option value="2">6</option>
+                            <option value="1">7</option>
+                            <option value="0">8</option>
+                        </select>
+                    </div>
+                </div>
+
+                <!--Marital Status-->
+                <div class="field">
+                    <label> Marital Status</label>
+                    <select class="ui dropdown">
+                        <option value="">Marital Status</option>
+                        <option value="4">Single</option>
+                        <option value="3">Married</option>
+                        <option value="2">Seprated</option>
+                        <option value="1">Divorced</option>
+                        <option value="0">Widowed</option>
+                    </select>
+                </div>
+
+
+
+                <h4 class="ui dividing header">Service Eligibility</h4>
+                <!--Service Status-->
+                <div class="field">
+                    <label>Service Status</label>
+                    <select class="ui dropdown">
+                        <option value="">Service Status</option>
+                        <option value="1">Approved</option>
+                        <option value="0">Banned</option>
+                    </select>
+                </div>
+
+                <!--Services Eligable For-->
+                <div class="field">
+                <div class="ui checkbox">
+                    <input type="checkbox" name="example">
```

Comment:
name should not be example.
Also, we're doing a single checkbox for all this, not multiple.
Just "Has ncessary paperwork" or the like.